src/pages/Profile/UserProfile.js

```diff
@@ -7,6 +7,7 @@ import Button from "@mui/material/Button";
 import { useTheme } from "@mui/material/styles";
 import { useTranslation } from "react-i18next";
 import UserInfo from "../../components/profile/UserInfo";
+import Header from "../../layouts/Header.js";
```

Comment:
Out of curiousity! Why do we need .js extension for this, but not the other imports?

Reply:
I think it doesn't matter

Reply:
removed